domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -96,6 +96,8 @@
   public static final String     DEFAULT_SCHEMA           = "public";
   public static final String     DEFAULT_JSONB_FIELD_NAME = "jsonb";
 
+  static final long              EXPLAIN_QUERY_THRESHOLD_DEFAULT = 1000;
```

Comment:
Can you add some javadoc explaining the unit (seconds?) and whether there is a magic value (-1?) to disable it.